### ISSUE TITLE
All IMG tags must have alt text

### ISSUE THREAD

**Issue by jsuddsjr** [pnp/generator-teams#88]
**Describe the bug**
Generated HTML files are not passing webhint inspections. 
* The HTML tag must have lang attribute
* IMG tags must have alt attribute.

**To Reproduce**
1. Generate default app.
2. Open any HTML file (such as index.html, privacy.html, or tou.html).

**Expected behavior**
Examples should demonstrate best coding and accessibility practices. 

**Screenshots**
Missing alt text.

```
<img src="https://static2.sharepointonline.com/files/fabric/assets/brand-icons/product-fluent/svg/teams_48x1.svg" 
class="logo">
```

Should be:

```
<img src="https://static2.sharepointonline.com/files/fabric/assets/brand-icons/product-fluent/svg/teams_48x1.svg" 
alt="Teams logo"
class="logo">
```

**Detailed information**
 - Generator version: [eg 2.5.0]
 - OS: [eg, Windows, MacOS]
 - Node version: [eg 8.10.0]
 - Npm version: [eg. 6.9.0]

**Additional information**

**Comment by jsuddsjr** [comment]
**Detailed information**

Generator version: 2.11.1
OS: Windows 10
Node version: 12.9.0
Npm version: 6.11.3

**Comment by wictorwilen** [comment]
Thanks for reporting this.

**Comment by wictorwilen** [comment]
Enhancement added in `2.15.0-preview`
